### PR TITLE
[Documentation] DICOM::Fields library perldocification

### PR DESCRIPTION
**Pull request by cmadjar**
Using perldoc/perlpod to document `Fields.pm` so that users can type in the terminal
`perldoc Fields.pm` and get the documentation.

Using the `pod2markdown` tool from perl `Pod::Markdown` module, the `Fields.md` file has been created so that we have a web displayed version of the documentation.
`pod2markdown Fields.pm > Fields.md`

Dependencies added: perldoc, Pod::Markdown, Pod::Usage

### CODE REVIEW

File: dicom-archive/DICOM/Fields.pm [hunk 1]
```diff
@@ -1,9 +1,62 @@
...
+      my ($group, $elem, $code, $numa, $name) = split(/\s+/, $line);
+      my @lst = ($code, $name);
+      $dict{$group}{$elem} = [@lst];
+    }
+  }
+
+
+=head1 DESCRIPTION
+
+Definitions of fields of DICOM headers. Not for medical use. This file is
```
Comment (nicolasbrossard) [inline dicom-archive/DICOM/Fields.pm]: Not for medical use? What do you mean by that?

File: dicom-archive/DICOM/Fields.pm [hunk 2]
```diff
@@ -1,9 +1,62 @@
...
+=head1 SYNOPSIS
+
+  use DICOM::Element;
+  use DICOM::Fields;	# Standard header definitions.
+  use DICOM::Private;	# Private or custom definitions.
+
+  # Initialize dictionary.
+  # Read the contents of the DICOM dictionary into a hash by group and element.
+  # dicom_private is read after dicom_fields, so overrides common fields.
+  BEGIN {
```
Comment (nicolasbrossard) [inline dicom-archive/DICOM/Fields.pm]: Not a big fan of this BEGIN block. I think this should be refactored (see review of #232 ) at a later time.

Comment (cmadjar) [reply dicom-archive/DICOM/Fields.pm]: agreed

File: dicom-archive/DICOM/Fields.pm (hunk 1 above)
Comment (cmadjar) [reply dicom-archive/DICOM/Fields.pm]: I did not write this. It was the available description before formatting so I did not edit it. Not sure it is really relevant so I'll remove it.

**Pull request merged**